worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -677,11 +678,18 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
   override def onResume(moduleName: String): Unit = {}
 
   override def onTrim(): Unit = {
-    actionService.submit(new Runnable {
-      override def run(): Unit = {
-        flushFileWriters()
-      }
-    })
+    if (trimInProcess.compareAndSet(false, true)) {
```

Comment:
Shall we also add the similar logic in ChannelsLimiter? Maybe we should put trimInProcess in MemoryPressureListener then.

Reply:
ChannelLimiter's onTrim isn't blocking operation.

Reply:
> ChannelLimiter's onTrim isn't blocking operation.

I'm not sure about the cost of frequently calling  ChannelLimiter.onTrim

Reply:
> As far as I know that this PR didn't change channel limiters' frequency.

I think the last pr changed this, we need a new pr to discuss and change this.

Reply:
I will merge this first, then I will find some unhealthy workers and then provide a fire flame about the cost during trim action. WDYT?

Reply:
> I will merge this first, then I will find some unhealthy workers and then provide a fire flame about the cost during trim action. WDYT?

SGTM, thanks!